rust/main/chains/hyperlane-ethereum/src/tx.rs

```diff
@@ -26,20 +26,26 @@ use tracing::{debug, error, info, instrument, warn};
 
 use crate::{EthereumMailboxCache, EthereumReorgPeriod, Middleware, TransactionOverrides};
 
-/// An amount of gas to add to the estimated gas
-pub const GAS_ESTIMATE_BUFFER: u32 = 75_000;
+/// An amount of gas to add to the estimated gas limit
+pub const GAS_LIMIT_BUFFER: u32 = 75_000;
 
-// A multiplier to apply to the estimated gas, i.e. 10%.
-pub const GAS_ESTIMATE_MULTIPLIER_NUMERATOR: u32 = 11;
-pub const GAS_ESTIMATE_MULTIPLIER_DENOMINATOR: u32 = 10;
+// A multiplier to apply to the estimated gas limit, i.e. 10%.
+pub const GAS_LIMIT_MULTIPLIER_NUMERATOR: u32 = 11;
+pub const GAS_LIMIT_MULTIPLIER_DENOMINATOR: u32 = 10;
+
+// A multiplier to apply to the estimated gas price, i.e. 10%.
+pub const GAS_PRICE_MULTIPLIER_NUMERATOR: u32 = 11;
```

Comment:
wonder if we should make this configurable?

Reply:
added to the `transactionOverrides` but still defaulting to these